openupgrade_scripts/apriori.py

```diff
@@ -1,9 +1,45 @@
-""" Encode any known changes to the database here
-to help the matching process
-"""
+import logging
+
+import configparser as ConfigParser
+
+import odoo
+
+def load_custom_config():
+    openupgrade_config_path = odoo.tools.config.misc.get("openupgrade").get("config_path")
```

Comment:
Hi. @jcdrubay. Thanks for your contribution.

I don't think that having 2 configuration files is a good thing.
- it makes the code to maintain much heavier
- it is not in the current design of Odoo (or OCA) to have several configuration files.

If in your custom configuration, you have multiple configuration files, you can use ``read()`` function to read multiple files and then ``write`` to write a single complete complete file.
https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.read

Reply:
I think that there is a misunderstanding. For the custom configuration, there would only be one file.

In total, there would be only 2 configuration files

- odoo.cfg: which is usually not committed and which is the standard configuration file of Odoo with (optionally) a single path `openupgrade_config_path`.
- openupgrade.cfg: only one file. This file should be committed together with the custom code being upgraded with several (optional) sections `renamed_modules` and `merged_modules`

Reply:
Note that the `openupgrade_config_path` is optional and that this file would only be used to declare necessary renaming/merging for custom code.
Inside the `odoo.cfg` file there would be no other addition than the configuration openupgrade_config_path. The listing of modules to rename and merge from non-custom code is being kept in the same place in the `apriori.py` file.